# Package index

## All functions

- [`.count_alleles()`](https://bitarellolab.github.io/balselr/reference/dot-count_alleles.md)
  : Count derived or ancestral alleles for a given vcf position
- [`.split_geno()`](https://bitarellolab.github.io/balselr/reference/dot-split_geno.md)
  : Split a genotype column from a vcf file
- [`.vcf_ncd1()`](https://bitarellolab.github.io/balselr/reference/dot-vcf_ncd1.md)
  : Make input file for NCD1 from VCF
- [`.vcf_ncd2()`](https://bitarellolab.github.io/balselr/reference/dot-vcf_ncd2.md)
  : Make input file for NCD2 from VCF
- [`ncd1()`](https://bitarellolab.github.io/balselr/reference/ncd1.md) :
  Calculate non-central statistic (NCD1)
- [`ncd1_input`](https://bitarellolab.github.io/balselr/reference/ncd1_input.md)
  : Input data to run ncd1 example
- [`ncd2()`](https://bitarellolab.github.io/balselr/reference/ncd2.md) :
  Calculate non-central statistic (NCD2)
- [`ncd2_input`](https://bitarellolab.github.io/balselr/reference/ncd2_input.md)
  : Input data to run ncd2 example
- [`parse_vcf()`](https://bitarellolab.github.io/balselr/reference/parse_vcf.md)
  : Parse vcf
- [`read_vcf()`](https://bitarellolab.github.io/balselr/reference/read_vcf.md)
  : Read vcf
- [`vcf_obj`](https://bitarellolab.github.io/balselr/reference/vcf_obj.md)
  : An example VCF (variant call format) read into R
